.github/workflows/test.yml

```diff
@@ -13,7 +13,6 @@ jobs:
       - uses: actions/checkout@v2
       - run: |
           npm install
-          npm run prepare-deps
```

Comment:
just a note on this one, `prepare-deps` is not merged with `all`. This is not so developer friendly, as now it will be longer to run `all` target. 

However, it's good that there is no couple between steps. Still ok for me :)

Reply:
I decided to put it to all, because now we have dependabot which updates the dependencies and it's better to re-build all necessary dependencies. And it take just a couple os seconds longer.